src/couch/src/couch_bt_engine.erl

```diff
@@ -192,6 +193,44 @@ handle_db_updater_info({'DOWN', Ref, _, _, _}, #st{fd_monitor=Ref} = St) ->
     {stop, normal, St#st{fd=undefined, fd_monitor=closed}}.
 
 
+format_status(normal, [_PDict, #st{} = St]) ->
+    [{data, [{"State", St}]}];
+
+format_status(terminate, [_PDict, #st{} = St]) ->
+    [{data, [{"State", reduce_record(St)}]}].
+
+
+reduce_record(#st{} = St) ->
```

Comment:
I think `redact` is a better word as we have a quite different meaning for `reduce` in couchdb.

Reply:
The `format_status/`2 was added to prevent huge btree structures hitting logs. Since the reduction of terms is not longer a goal I am thinking about removing this function.